README.md

```diff
@@ -101,6 +101,13 @@ For example, to install dependencies on Arch Linux:
 sudo pacman -S --needed glib2 libgexiv2 meson vala gstreamer gst-plugins-base-libs gdk-pixbuf2 gobject-introspection gst-plugins-good gst-plugins-bad gst-plugin-va
 ```
 
+
+To install dependencies on Debian/Ubuntu:
+
+```bash
+apt install valac libgexiv2-dev libgstreamer1.0-dev libgdk-pixbuf-2.0-dev libgdk-pixbuf-2.0-0 gobject-introspection help2man valadoc
```

Comment:
Have you verified your list? This project cannot be built without meson, but meson is not included in your list of dependencies.